src/sections/Community/Handbook/repository.js

```diff
@@ -236,6 +236,7 @@ const Repository = () => {
                               >
                                 <img className="github-icon" src={github} alt="github-icon" />
                               </a>
+                              <div className="accessRequired"> { accessRequired != "" ? accessRequired : ""}</div>
```

Comment:
??

Reply:
@H-A-R-S-H-K explain this?

Reply:
@sudhanshutech the reason why I made such changes is because the frontend projects subdata follows this type of code to display status of frontend projects such as "*program participation required" on https://layer5.io/community/handbook/repository-overview page.